release.yaml

```diff
@@ -258,5 +258,5 @@ repositories:
         ref: v0.15.4
```

Comment:
@Gerrit91 
This can be removed ?

Reply:
No, this is our initial ETCD, it's the most important one.

Reply:
I see